CHANGELOG.md

```diff
@@ -0,0 +1,23 @@
+# Changelog (Pypi package)
+
+## [1.3.44] 2022-01-19
+
+### Changed
+
+Some DataStats properties changed in the naming and some of them was added, see [HERE](https://github.com/ToucanToco/toucan-connectors/commit/9d74efb6a6e37c6fbcd951743ac418aa84911704) for more informations.
+
+### Fixed
+
+- Fixes on sql/snowflake (don't run count for DESCRIBE or SHOW queries + don't use -1 as default rows count)
+- Fixes on sharepoint and onedrive connectors.
+
+## [1.3.43] 2022-01-17
+
+### Added
+
+- Added filenames_to_match param to extract multiple files on connectors sharepoint and onedrive.
+- Added a dev container for developping safely on connectors.
```

Comment:
It's nice, but I'm not sure we're gonna be good devs and maintain this changelog properly with every new feature added/changed or bug fixed.
I think it's more reasonable to commit on having one entry for each major version, indicating what to do regarding the breaking changes.
What do you think ? (Maybe I'm a bit too pessimistic there ahah)

Reply:
😆 i think we can give a try a least !